web_responsive_pwa/static/src/js/pwa_install.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('web_responsive_pwa.systray.install', function (require) {
+"use strict";
+
+var core = require('web.core');
+var session = require('web.session');
+var SystrayMenu = require('web.SystrayMenu');
+var Widget = require('web.Widget');
+var QWeb = core.qweb;
+
+if ('serviceWorker' in navigator) {
+  window.addEventListener('load', () => {
```

Comment:
Sorry, only ES5 syntax is allowed for v12. Please make linters happy: https://travis-ci.org/github/OCA/web/jobs/688422849#L805-L806

This applies to the whole file.

Reply:
Done, thanks